### ISSUE TITLE
dashboard not accessible without mouse if navigation is closed

### ISSUE THREAD

**Issue by CarolingerSeilchenspringer** [liqd/a4-meinberlin#3480]
**URL:** dashboard
**user:** initiator
**expected behaviour:** I can navigate through my dashboard without mouse
**behaviour:** if navigation tabs (both project and modules) are closed I cannot access them without mouse. I have to click for each point of the closed menu (and can even open the according form) but cannot see them as menu doesn't open.
**important screensize:**
**device & browser:** tested on big sur, chrome and firefox
**Comment/Question:** 

Screenshot?

**Comment by CarolingerSeilchenspringer** [comment]
@phillimorland There is no focus on the arrow. The focus is on "alle Beteiligungsprojekte", then nowhere (should be the arrow) and then on the top of the left navigation where project name is shown. Can we make it more clear?
